napari/_qt/qt_main_window.py

```diff
@@ -89,6 +90,12 @@ def __init__(self, qt_viewer: QtViewer, parent=None) -> None:
         if SETTINGS.plugins.call_order is not None:
             plugin_manager.set_call_order(SETTINGS.plugins.call_order)
 
+        else:
+            # need to set the call_order for plugin_dialog to work
```

Comment:
quick comment here on what "work" means?  (like what would happen if this wasn't here?)

Reply:
@tlambert03 thanks for catching that!  It is no longer needed.  I think I used to pass in SETTINGS.plugins.call_order somewhere.  I removed this part.